src/fu-util.c

```diff
@@ -57,6 +57,7 @@ struct FuUtilPrivate {
 	gboolean		 assume_yes;
 	gboolean		 sign;
 	gboolean		 show_all_devices;
+	gboolean		 disable_ssl_strict;
```

Comment:
Since `fwupdtool` can actually download files too these days, I think the same option should be ported to it as well (and it's `bash-completion` too)

Reply:
And maybe `fwupdagent`? not sure.

Reply:
I've done fwupdtool, but I don't think fwupdagent should be done. If you can't get your corporate proxy doing SSL properly, you shouldn't be able to use the agent.